src/App.tsx

```diff
@@ -1,14 +1,55 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Filters } from './types/Filter';
 
-const USER_ID = 0;
+const USER_ID = 10908;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+  const [filter, setFilter] = useState(Filters.ALL);
+
+  const removeError = () => {
+    setErrorMessage(null);
+  };
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((todosFromServer: Todo[]) => {
+        setTodos(todosFromServer);
+      })
+      .catch((error) => {
+        setErrorMessage(error.message);
+        setTimeout(removeError, 3000);
+      });
+  }, []);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const uncompletedTodos = todos.filter(todo => !todo.completed);
+  const completedTodos = todos.filter(todo => todo.completed);
+
+  const filterTodos = () => {
+    switch (filter) {
+      case Filters.ACTIVE:
+        return uncompletedTodos;
+
+      case Filters.COMPLETED:
+        return completedTodos;
+      default:
+        return todos;
+    }
+  };
+
+  const filteredTodos = filterTodos();
```

Comment:
is it necessary?😁 u just can rename your `filterTodos` to `filteredTodos` and use it in props in TodoList